src/javascript/app/common/redirect_banner.js

```diff
@@ -12,14 +13,30 @@ const RedirectBanner = (() => {
             const eu_country = isEuCountrySelected(Client.get('residence')) || isEuCountrySelected(State.getResponse('website_status.clients_country'));
             if (eu_country) {
                 handleRedirect();
+            } else {
+                handleRedirectROW();
             }
             
         });
 
     };
 
     const handleRedirect = () => {
-        window.location.href = '/move-to-deriv/';
+        window.location.href = 'https://www.binary.bot/movetoderiv.html';
```

Comment:
/move-to-deriv/

Reply:
Hi @habib-deriv @meenakshi-deriv the new one is  `/binary-to-deriv/` which we need to append to the url. We won’t need any redirection I think. 🤔 

`binary.com/binary-to-deriv/`